drud/ddev
# webcache deprecation, fixes #1495

## The Problem/Issue/Bug:

We're no longer supporting the experimental webcache feature (macOS only)

## How this PR Solves The Problem:

* Make it less obtrusive in the config.yaml (don't show it unless it's turned on)
* Add minor docs about it in the comment section of config.yaml
* Turn off webcache-specific testing except for the one TestWebcache test.
* Make clear in docs (readthedocs) that it's deprecated and what it might be used for.

## Manual Testing Instructions:

* `ddev config` with `webcache_enabled: false` should make it disappear from the main top section of config.yaml
* `ddev config` should create an explanatory section at bottom of config.yaml
* Read docs (https://github.com/rfay/ddev/blob/c0746749649f004e22a5c7f3afbbab4d49db9f37/docs/users/performance.md#using-webcache_enabled-to-cache-the-project-directory-deprecated-macos-only)

## Related Issue Link(s):

OP #1495 
@kaystrobach's #1528 PR

## Release/Deployment notes:
